config/packages/enqueue.yaml

```diff
@@ -2,3 +2,5 @@ enqueue:
     transport:
         default: '%env(ENQUEUE_DSN)%'
     client: ~
+    extensions:
+        doctrine_ping_connection_extension: true
```

Comment:
?

Reply:
Tests connection to the database, and reconnects if it is lost.

Reply:
For queues

Reply:
okay